app/controllers/issues.js

```diff
@@ -9,17 +10,14 @@ export default Controller.extend({
 
   navLinks: service(),
 
-  queryInput: computed('query', function() {
-    return this.get('query');
-  }),
+  queryInput: computed.reads('query'),
```

Comment:
A project that I worked on recently, the linter asked me to import `reads` directly from `@ember/object/computed`. I was surprised that this worked without a lint error. (No need to change it. Just an observation.)